### PR TITLE
example celery module; flask celery status server; test for it

### PR DESCRIPTION
**Pull request by PeterDSteinberg**
celery_status_server.py  runs flask server on port 5070.  URLs:

#### Register a job with email, id, callback
/register_job?email=me@email.com&ticket_id=abc&callback=somewhereelse.com/callback
#### Get all statuses
/celery_status_server
#### Example async delay function
/example_async

test_celery_status_server.py tests the server above when it is running.  Requires celery and redis running.

example_async.py is just an example celery app with timer sleep function.

### CODE REVIEW

File: celery_status_server.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+@server_app.route('/example_async', methods=['GET'])
+def example():
+
+    ticket_id =  example_async.example_async.delay()
+    print('ticket_id', ticket_id)
+    url = '{0}/register_job?email={1}&ticket_id={2}&callback="http://google.com"'.format(server_url, ticket_id, ticket_id)
+    return json.dumps({'ticket_id': str(ticket_id)})
+
+@server_app.route("/register_job", methods=['GET'])
```
Comment (talumbau) [inline celery_status_server.py]: I think this should be a POST, right?

Comment (PeterDSteinberg) [inline celery_status_server.py]: Yes that's a better idea.  Will change and move soon.

**Comment by talumbau** [comment]
Before moving on though, let's close this PR and open on webapp-public

File: celery_status_server.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                                                         'eta': eta,
+                                                         }
+                    else:
+                        pass
+    for key in TICKET_IDS_WATCHED:
+        if key not in return_val:
+            return_val[key] = {'inputs': TICKET_IDS_WATCHED[key].copy(),
+                               'status': 'done'}
+            to_pop.append(key)
+    response = {}
```
Comment (PeterDSteinberg) [inline celery_status_server.py]: It should be possible here to call AsyncResult() to see if the job is really done or if the ticket id was never processed.  Currently either of those cases would show up here as 'done' status.

**Pull request closed without merging**